w-books/src/App/components/Header/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import wlogo from '../../../assets/Header/wbooks_logo.svg';
+import icon_notifications from '../../../assets/Header/notifications.svg';
+import icon_add_book from '../../../assets/Header/add_book.svg';
+import icon_user from '../../../assets/Header/user_dummy.png';
+
+class Header extends Component {
```

Comment:
if this component hasnt internal states or function to handle, you can declare it like a function.

```
function Header () {
...
..
...
}

export default Header;
```